platform/admin/core/admin-core-insecuredefaults/src/test/java/org/codice/ddf/admin/insecure/defaults/service/KeystoreValidatorTest.java

```diff
@@ -161,8 +161,8 @@ public void testBlacklistKeystoreDoesNotExist() throws Exception {
                                 DEFAULT_KEYSTORE_PASSWORD),
                         String.format(KeystoreValidator.BLACKLIST_KEYSTORE_DOES_NOT_EXIST_MSG,
                                 path,
-                                FAKE_BLACK_LIST,
-                                FAKE_BLACK_LIST)};
+                                Paths.get(FAKE_BLACK_LIST),
```

Comment:
See if we can convert the `FAKE_BLACK_LIST` field to a `Path` instead of having to do the conversion every time.

Check the other constants as well.